### PR TITLE
[tensorflow]|[test] copy tfs-mme tests to dlc repo

### PR DESCRIPTION
**Pull request by ChuyangDeng**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

*Description:*
Copy TFS sagemaker MME local integration tests to DLC repo

*Tests run:*
Local integration tests on [sagemaker-tensorflow-serving-container](https://github.com/aws/sagemaker-tensorflow-serving-container)

*DLC image/dockerfile:*
tensorflow/inference/1.15.3/py3,py36
tensorflow/inference/2.2/py3

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by arjkesh** [comment]
@Satish615 could you take a look at this PR, as it might have conflicts with your local mode test PR?

**Comment by Satish615** [comment]
@ChuyangDeng The current code build jobs are not running the local mode sagemaker tests. Did you run the local tests manually on these DLC Images? if not can you please hold on until #334  is merged

**Comment by ChuyangDeng** [comment]
> @ChuyangDeng The current code build jobs are not running the local mode sagemaker tests. Did you run the local tests manually on these DLC Images? if not can you please hold on until #334 is merged

Yes, I can wait until #334 

The local integrations tests were run in this repo: https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/buildspec.yml#L23

Also, what is the sanity check that is failing? I cannot see the error message.

**Comment by Satish615** [comment]
> > @ChuyangDeng The current code build jobs are not running the local mode sagemaker tests. Did you run the local tests manually on these DLC Images? if not can you please hold on until #334 is merged
> 
> Yes, I can wait until #334
> 
> The local integrations tests were run in this repo: https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/buildspec.yml#L23
> 
> Also, what is the sanity check that is failing? I cannot see the error message.

Sanity check is failing for rsa package.  we are addressing in a different pr. 
#334 is merged now. Can you rebase on and run the tests again.

File: tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -4,6 +4,7 @@ LABEL maintainer="Amazon AI"
 # Specify LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```
Comment (TusharKanekiDey) [inline tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu:7]: Do we need the similar line in GPU dockerfiles as well ?

Comment (ChuyangDeng) [reply tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu:7]: MME is not supported to GPU yet.

**Comment by TusharKanekiDey** [comment]
Build for TF2 is failing here.(https://github.com/aws/deep-learning-containers/blob/master/tensorflow/inference/docker/2.2.0/py3/Dockerfile.cpu#L99). Might need to update the dockerfile as well.

**Comment by ChuyangDeng** [comment]
The tests that's skipping GPU are due to the reason that GPU containers does not support MME yet.

**Pull request merged**